packages/template-retail-react-app/app/partials/product-view/index.jsx

```diff
@@ -104,11 +104,14 @@ const ProductView = ({
         stepQuantity
     } = useProduct(product)
     const canAddToWishlist = !isProductLoading
-    const canOrder =
-        !isProductLoading &&
-        variant?.orderable &&
-        parseInt(quantity) > 0 &&
-        parseInt(quantity) <= stockLevel
+    
+    // @@@ Skip this check as it doesn't entire make sense for standard products.
+    const canOrder = true
+    // const canOrder =
+    //     !isProductLoading &&
+    //     variant?.orderable &&
+    //     parseInt(quantity) > 0 &&
+    //     parseInt(quantity) <= stockLevel
```

Comment:
Standard products need a different conditional to determine whether they can be ordered.